.github/workflows/build-test.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         os: [ubuntu-latest]
         python: ["3.11"]
         mlpackage: [lightgbm, xgboost, tensorflow, pytorch]
+      fail-fast: false
```

Comment:
This allows the otherml packages to complete even if one fails. I think this makes sense, given that each package is likely to fail for its own reasons